internal/controlplane/handlers_oauth_test.go

```diff
@@ -313,15 +314,14 @@ func TestGetAuthorizationURL(t *testing.T) {
 			}
 			mockJwt := mockjwt.NewMockJwtValidator(ctrl)
 
-			server, err := NewServer(
-				store,
-				evt,
-				c,
-				mockJwt,
-				&ratecache.NoopRestClientCache{},
-				providers.NewProviderStore(store),
-			)
-			require.NoError(t, err, "failed to create server")
+			server := &Server{
```

Comment:
Have avoided using `NewServer` in the tests because it would involve passing a lot of `nil`s (or creating mocks unnecessarily)